### PR TITLE
fix: responsiveObserve init miss sizing

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

ref #20356

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Fix responsive related component (like Grid) missing sizing when init.       |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by zombieJ** [comment]
Hmmm...... CI failed...

File: components/_util/responsiveObserve.ts [hunk 1]
```diff
@@ -77,6 +77,8 @@ const responsiveObserve = {
         mql,
         listener,
       };
+
+      listener(mql);
```
Comment (afc163) [inline components/_util/responsiveObserve.ts]: 这样就好了？

Comment (zombieJ) [reply components/_util/responsiveObserve.ts]: 嗯，初始化的时候就有一个默认值。之后 add 就是新的。 test case 里要调整一下。

Comment (afc163) [reply components/_util/responsiveObserve.ts]: 给力，测试代码也清晰多了。

**Pull request merged**